app/src/main/java/fr/free/nrw/commons/utils/DateUtils.java

```diff
@@ -33,4 +35,10 @@ public static String getTimeAgo(Date currDate, Date itemDate) {
             return "0-seconds";
         }
     }
+
+    public static String getCurrentDate() {
+        SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd", Locale.US);
```

Comment:
Should this Locale always be US?

Reply:
Yes, as the template string (eg. `Template:Potd/2015-08-02`) needs to be in this [format](https://www.mediawiki.org/wiki/API:Showing_interesting_content) irrespective of what locale the user is in.